### PR TITLE
Add feature

### PR DESCRIPTION
**Pull request by alexmedinasf**
Copy webpack folders from webpack exercise
Created html 
Basic style with CSS
Basic functionality with js

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# Leaderboard
\ No newline at end of file
+# Leaderboard
+
```
Comment (cynthiainga) [inline README.md:2]: - [ ] Please kindly make your `README` file more professional, you can use this [templante](https://github.com/microverseinc/readme-template/blob/master/README.md) provided by Microverse 🙃 🙃

File: package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Comment (cynthiainga) [inline package.json]: - [ ] When running the command `npm run start` the project doesn’t open in the browser. You miss some scripts in your `package.json` file, please kindly make sure you setup your project correctly 🙃 🙃

Your `scripts` should be similar to the example below 👇 👇

```json
  "scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
  }
```

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @alexmedinasf, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Used correct Gitflow ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- I noticed that your commit messages are not capitalized, but the commit message should be written in the imperative tense and capitalized. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8), please kindly remember this in your next commits messages 🙃 😉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (mgmediaweb) [reply README.md:2]: Good job following the advice of the previous reviewer

File: package.json (hunk 2 above)
Comment (mgmediaweb) [reply package.json]: Kindly, I suggest you review this topic carefully, your app does not start with the traditional "npm run start" command, check the configuration of your package.json file or replace it with a correct one.

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @alexmedinasf , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

work4

 ## Highlights 💯 
- Good job following the advice of the previous reviewer ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by uwadonat — APPROVED** [review]
## Hi @alexmedinasf,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
### Highlights
- No linter errors ✔️
- Organized project ✔️
approval


# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**